docs/sources/manage-users/server-admin-manage-users.md

```diff
@@ -0,0 +1,94 @@
++++
+title = "Server Admin - Manage users"
+type = "docs"
+weight = 100
++++
+
+# Manage users as a Server Admin
+
+This topic explains user management tasks performed by Grafana Server Admins.
+
+In order to perform any of these tasks, you must be logged in to Grafana on an account with Grafana Server Admin permissions. For more information about Grafana Admin permissions, refer to [Grafana Server Admin role]({{< relref "../permissions/_index.md#grafana-server-admin-role" >}})
+
+> **Note:** The Grafana Server Admin role does not exist in Grafana Cloud. Grafana Cloud users cannot perform tasks listed in this section.
+
+## View the user account list
+
+See a complete list of users with accounts on your Grafana server.
+
+1. Hover your cursor over the **Server Admin** (shield) icon until a menu appears.
+1. Click **Users**.
+
+{{< docs/shared "view-user-list" >}}
+
+Grafana displays all user accounts on the server, listed in alphabetical order by user name. The following information is displayed:
+- **Login -** The value in the **Username** field of the account.
+- **Email -** The email associated with the user account.
+- **Name -** The value in the **Name** field of the account.
+- **Seen -** How long ago the user logged in.
+- **Server Admin status -** If the user account has **Grafana Admin** option set, then a shield icon is displayed.
+
+## View user account details
+
+See all details associated with a specific user account.
+
+1. [View the user account list](#view-the-user-account-list).
```

Comment:
Can we add text here and not just a link?

Reply:
I am trying to do this with single sourcing.